pkg/apis/pravega/v1alpha1/pravegacluster_types.go

```diff
@@ -78,6 +77,8 @@ type ClusterSpec struct {
 	// https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md
 	TLS *TLSPolicy `json:"tls,omitempty"`
 
+	Authentication *AuthenticationParameters `json:"authentication,omitempty"`
```

Comment:
Better to have a few comments for `Authentication` like any other fields in the struct.

Reply:
Done.